src/components/Calculator.jsx

```diff
@@ -14,6 +14,8 @@ export const Calculator = () => {
     result: null,
     operation: "sum",
   });
+  const [history, setHistory] = React.useState([]);
+  const [historyPos, setHistoryPos] = React.useState(0);
```

Comment:
Use history.length instead of a new useState

Reply:
now the history.length is used